tests/5.0/target/test_target_device1.c

```diff
@@ -0,0 +1,53 @@
+//===------test_target_device.c--------------------------------------===//
+//
+// OpenMP API Version 5.0 Nov 2018
```

Comment:
This seems to be a 4.5 test, not a 5.0 test

Reply:
Moved the file to 4.5/target directory...